shell/src/main/java/tachyon/shell/command/CreateLineageCommand.java

```diff
@@ -91,4 +91,9 @@ public void run(String... args) throws IOException {
     }
     System.out.println("Lineage " + lineageId + " has been created.");
   }
+
+  @Override
+  public String getUsage() {
+    return "createLineage <inputFiles,...> <outputFiles,...> <cmd>";
```

Comment:
I think this should be `<inputFile1,...> <outputFile1,...> [<cmd_arg1> <cmd_arg2> ...]`